.github/labeler.yml

```diff
@@ -1,42 +1,54 @@
 # For reference, please look at https://github.com/actions/labeler
 
 GitHub Actions:
-- .github/**/*
+- changed-files:
+  - any-glob-to-any-file: '.github/**/*'
 
 Price addition:
-- 'src/views/AddPriceHome.vue'
-- 'src/views/AddPriceSingle.vue'
-- 'src/views/AddPriceMultiple.vue'
+- changed-files:
+  - any-glob-to-any-file: 'src/views/AddPriceHome.vue'
```

Comment:
shouldn't it be 1 line with an array of values ?

https://github.com/actions/labeler#basic-examples

Reply:
it works with my syntax on other repos 🤷